src/main/java/picard/sam/SamErrorMetric/CollectSamErrorMetrics.java

```diff
@@ -328,21 +347,41 @@ private void writeMetricsFileForAggregator(final BaseErrorAggregation locusAggre
     }
 
     /**
-     * Advance the iterator until at or ahead of locus. if there's a variant at the locus return true, otherwise false.
+     * If there's a variant at the locus return true, otherwise false.
      * <p>
-     * HAS SIDE EFFECTS!!! draws from vcfIterator!!!
+     * HAS SIDE EFFECTS!!! Queries the vcfFileReader
      *
-     * @param vcfIterator        a {@link VariantContext} iterator to advance (assumed sorted)
-     * @param locus              a {@link Locus} at which to examine the variants
-     * @param sequenceDictionary a dictionary with which to compare the Locatable to the Locus...
+     * @param vcfFileReader      a {@link VCFFileReader} to query for the given locus
+     * @param locusInfo          a {@link SamLocusIterator.LocusInfo} at which to examine the variants
      * @return true if there's a variant over the locus, false otherwise.
      */
-    private static boolean advanceIteratorAndCheckLocus(final PeekableIterator<VariantContext> vcfIterator, final Locus locus, final SAMSequenceDictionary sequenceDictionary) {
-        while (vcfIterator.hasNext() && (vcfIterator.peek().isFiltered() ||
-                CompareVariantContextToLocus(sequenceDictionary, vcfIterator.peek(), locus) < 0)) {
-            vcfIterator.next();
+    private static boolean checkLocus(final VCFFileReader vcfFileReader, final SamLocusIterator.LocusInfo locusInfo) {
+
+        boolean overlaps = false;
```

Comment:
why so many empty lines?

Reply:
I've found that leaving a little extra whitespace around pays dividends in readability.  

I'll remove a few.